tests/integration/config/plugins/integration-tests/data/tests/tests_common.txt

```diff
@@ -706,6 +706,21 @@ test "Depart and Ignore Warning Message"
 		call "Wait 55 steps"
 
 
+test "Depart and Abort on Warning Message"
+	status partial
+	description "Helper function to attempt departure from a planet, then stop when an expected warning message is given."
+	sequence
+		input
+			key d
+		# The game needs 2 gamesteps for processing the launch before commands like the jump command can be given.
+		# Using empty inputs to trigger those game loops.
+		input
+		input
+		# Acknowledge the expected message
```

Comment:
```suggestion
		# Ignore the expected message
```
this time you need it this way ^^
cancel works too, idk

Reply:
The first one is actually ignoring the warning, because it continues to take off. This accepts the warning and doesn't take off.
~~Time to add a way to customise the text in those buttons.~~